backend/webui_service/ui_service.go

```diff
@@ -1,42 +1,40 @@
 // SPDX-License-Identifier: Apache-2.0
 // Copyright 2024 Canonical Ltd.
 
-// +build ui
+//go:build ui
 
 package webui_service
 
 import (
-    "io/fs"
-    "net/http"
-    "strings"
+	"io/fs"
+	"net/http"
+	"strings"
 
-    "github.com/gin-gonic/gin"
-    "github.com/omec-project/webconsole/backend/logger"
-    "github.com/omec-project/webconsole/ui"
+	"github.com/gin-gonic/gin"
+	"github.com/omec-project/webconsole/backend/logger"
+	"github.com/omec-project/webconsole/ui"
 )
 
 func AddUiService(engine *gin.Engine) {
 	logger.WebUILog.Infoln("Adding UI service")
-    staticFilesSystem, err := fs.Sub(ui.FrontendFS, "frontend_files")
-    if err != nil {
-        logger.WebUILog.Fatal(err)
-    }
-
-    engine.Use(func(c *gin.Context) {
-        if !isApiUrlPath(c.Request.URL.Path){
-            htmlPath := strings.TrimPrefix(c.Request.URL.Path, "/") + ".html"
-            if _, err := staticFilesSystem.Open(htmlPath); err == nil {
-                c.Request.URL.Path = htmlPath
-            }
-            fileServer := http.FileServer(http.FS(staticFilesSystem))
-            fileServer.ServeHTTP(c.Writer, c.Request)
-            c.Abort()
-        }
-    })
+	staticFilesSystem, err := fs.Sub(ui.FrontendFS, "frontend_files")
+	if err != nil {
+		logger.WebUILog.Fatal(err)
+	}
+
+	engine.Use(func(c *gin.Context) {
+		if !isApiUrlPath(c.Request.URL.Path) {
+			htmlPath := strings.TrimPrefix(c.Request.URL.Path, "/") + ".html"
+			if _, err := staticFilesSystem.Open(htmlPath); err == nil {
+				c.Request.URL.Path = htmlPath
+			}
+			fileServer := http.FileServer(http.FS(staticFilesSystem))
+			fileServer.ServeHTTP(c.Writer, c.Request)
+			c.Abort()
+		}
+	})
 }
 
-func isApiUrlPath(path string) bool{
-    return strings.HasPrefix(path, "/config/v1/") || strings.HasPrefix(path, "/api/");
+func isApiUrlPath(path string) bool {
+	return strings.HasPrefix(path, "/config/v1/") || strings.HasPrefix(path, "/api/") || path == "/login" || path == "/status"
```

Comment:
We have a frontend path that's just `/login`. This should only be true if it's POST /login, not GET /login